config-controller/api/v1alpha1/policy_types.go

```diff
@@ -38,27 +38,39 @@ type EnforcementAction string
 type SecurityPolicySpec struct {
 	// +kubebuilder:validation:Required
 	// +kubebuilder:validation:Pattern=`^[^\n\r\$]{5,128}$`
-	PolicyName string `json:"policyName,omitempty"`
+	// PolicyName is the name of the policy as it appears in Central.  Note that changing this value will rename the policy in Central.  This field must be unique.
```

Comment:
We should just replicate the swagger doc here to explain the field. And then add more stuff to explain any more policy as code specific stuff.

Reply:
Do you mean just grab the doc from swagger and copy over by hand?  Or did you mean something more fancy like a script to keep them in sync?